client/my-sites/posts/post-list.jsx

```diff
@@ -277,7 +277,7 @@ var Posts = React.createClass( {
 				/>
 			);
 		} else {
-			if ( this.props.loading || ! this.props.sites.fetched ) {
+			if ( this.props.loading || ! this.props.haveSites ) {
```

Comment:
Previously, we wanted those placeholders to show while sites were still being fetched, and the 'no content' message if we were done fetching, and there weren't any posts. The trouble is that we don't have any way to tell if we are fetching or done with it for our Redux `sites`...